app/src/main/res/layout/fragment_social_links.xml

```diff
@@ -2,12 +2,8 @@
 <FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Change this framelayout to coordinator layout

Reply:
that was the bug if I changed it back then snackbar will take it as the first coordinate-Layout parent  and appear as before

Reply:
if you want cordirdinate layout in view hiarchy we could nest it inside frameLayout